master/reference/public-cloud/gce.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Calico on Google Cloud
+---
+
+Calico is designed to provide high performance massively scalable virtual
+networking for containers in public cloud environments such as
+Google Cloud.
+
+## Why Use Calico in Google Cloud
+
+#### Network Policy for Containers
+
+While Google Cloud Firewall Rules control network traffic sent to VM instances,
+it does not cover containers running on them. Calico automatically implements
+fine-grain, dynamic network policy for containers.
+
+#### No Overlays
```

Comment:
A stronger point would be that you can go past the GCE route limit using ipip, and you can get more flexible IPAM than a /24 per host.